### DISCUSSION TITLE
What language can I use if I got a 30% in photography?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#108347] [Programming Help]
### Body

Hi,

I got my report card, and it wasn't good, so I am starting to think I am much dumber than I thought and maybe can't do coding.

### Guidelines

- [x] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Ariffazeel99** [comment]
start with Mern stack, and for that, you can start with simple HTML and CSS, then PHP or JavaScriptto start with Mern stack, and for that, you can start with simple html and css then Php or javascript.
This will help you to get started with web development.
If you need any help you can always ping me, no worries.

here is the website: https://roadmap.sh/
You can easily find the road map of your required path.

**Reply by ghost** [reply]
How hard is it to move on from HTML., CSS and JavaScript which I already have used several times [My HTML code](https://github.com/PyDOS8?tab=repositories&q=&type=&language=html&sort=) and a project I written that uses Markdown, JavaScript, HTML and CSS. [PyTutorials](https://github.com/PyDOS8/pytutorials.github.io)

**Reply by Ariffazeel99** [reply]
It's not hard if you put in some patience and enjoy the learning process. I also started with Mern stack six months ago.

**Reply by Lanrad23** [reply]
Don't be so harsh on yourself. Just because you didn't do well in photography doesn't mean you can't be good at coding or anything else. Your grades in one subject don't decide how smart you are or what you can achieve in other areas.
To answer your question, you can begin by learning basic HTML and CSS.

**Comment by ghost** [comment]
> ### Body
> Hi,
> 
> I got my report card, and it wasn't good, so I am starting to think I am much dumber than I thought and maybe can't do coding.
> 
> ### Guidelines
> * [x]  I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

Hey @PyDOS8 if you are on github rn this means you're on the right track. Don't worry that you can make it. After all coding is not about grading.

If you are willing find path just do what you think that fits you, enjoy the phase and never stop learning.

I damn sure you can make it. All the best.

*Just an Advice*

**Comment by hossainalmahdi** [comment]
You can start with python. If you are new into coding. Start with python. Try to solve some interesting problems. No need to goo deeper at first. Just solve something simple by implementing logic. If you like that. Start Digging!
